okex-client/tests/client.rs

```diff
@@ -205,38 +225,22 @@ async fn get_positions() -> anyhow::Result<()> {
 
 #[tokio::test]
 async fn close_positions() -> anyhow::Result<()> {
-    if let Some(client) = demo_okex_client() {
+    if let Some(client) = demo_okex_client().await? {
         let instrument = OkexInstrumentId::BtcUsdSwap;
-        let margin = OkexMarginMode::Cross;
-        let position_side = OkexPositionSide::Net;
         let order_side = OkexOrderSide::Buy;
-        let order_type = OkexOrderType::Market;
+        let currency = "BTC".to_string();
```

Comment:
Will this every be a different currency?

- if we are always trading with `BTC` then we should not expose it in the interface.